pages/attend/travel.md

```diff
@@ -68,3 +68,20 @@ The US-RSE’24 conference will again be offering travel grants via an applicati
 - [Paseo del Bosque Trail](https://www.cabq.gov/parksandrecreation/open-space/lands/paseo-del-bosque-trail)
 - [Breweries](https://www.newmexico.org/things-to-do/cuisine/breweries-tap-rooms/)
 - [Railrunner to Santa Fe](https://www.visitalbuquerque.org/things-to-do/tours-sightseeing/self-guided/rail-runner/)
+
+## Parking and Transportation to the Venue
+
+US-RSE’24 will be taking place in the west complex of the Albuquerque Convention Center on floors 1 and 2. The registration desk will be on floor 2, outside ‘Ballroom A’.
```

Comment:
It's actually floors 1 and 3 - not 2.

Reply:
And Floor 3, outside Ballroom A.

Reply:
If you want to add the map... https://www.visitalbuquerque.org/meetings/convention-center/specs/